pkg/syncer/syncer.go

```diff
@@ -67,13 +75,34 @@ func StartSyncer(upstream, downstream *rest.Config, resources sets.String, clust
 		specSyncer.Stop()
 		return nil, err
 	}
+	leaseClient, err := clientset.NewForConfig(upstream)
+	if err != nil {
+		specSyncer.Stop()
+		return nil, err
+	}
+	leaseController := lease.NewController(
+		clock.RealClock{},
+		leaseClient,
+		fmt.Sprintf("cluster-%s", cluster),
+		int32(leaseDurationSeconds),
+		// TODO: onRepeatedHeartbeatFailure ? closeAllConns ?
+		nil,
+		time.Duration(float64(time.Duration(leaseDurationSeconds))*leaseRenewIntervalFraction),
+		// TODO: namespace ?
+		v1.NamespaceNodeLease,
```

Comment:
How about `syncer-system`? This is the namespace the syncer gets installed into in physical clusters, so this keeps us from having to think about two different namespace names.

Reply:
✅ 
What surprises me though is that leases are created in a namespace that does not exist 🤔 
```shell
$ k get lease -A -o wide
NAMESPACE       NAME        HOLDER      AGE
syncer-system   cluster-a   cluster-a   9s
syncer-system   cluster-b   cluster-b   11s
syncer-system   cluster-c   cluster-c   11s
```
```shell
$ k get ns
No resources found
```